components/Form/NewContactForm.js

```diff
@@ -0,0 +1,268 @@
+        type="text"
+        id="first_name"
+        name="first_name"
+        value={formData.first_name}
+        onChange={handleInputChange}
+        required
+      />
+
+      <label htmlFor="last_name">Last Name:</label>
+      <StyledInput
+        type="text"
+        id="last_name"
+        name="last_name"
+        value={formData.last_name}
+        onChange={handleInputChange}
+      />
+
+      <label htmlFor="birthday">Birthday:</label>
+      <StyledInput
+        type="date"
+        id="birthday"
+        name="birthday"
+        value={formData.birthday}
+        onChange={handleInputChange}
+      />
+
+      <label htmlFor="friendship_status">Friendship Status:</label>
+      <StyledSelect
+        id="friendship_status"
+        name="friendship_status"
+        value={formData.friendship_status}
+        onChange={handleInputChange}
+      >
+        <option value="set Status">Set status</option>
+        <option value="Friend">Friend</option>
+        <option value="Best Friend">Best Friend</option>
+        <option value="Gold Friend">Gold Friend</option>
+        <option value="Family">Family</option>
+        <option value="Sister">Sister</option>
+        <option value="Sister from another mister">
+          Sister from another mister
+        </option>
+        <option value="Brother">Brother</option>
+        <option value="Brother from an other mother">
+          Brother from an other mother
+        </option>
+        <option value="New">New</option>
+        <option value="I think about">i think about</option>
+      </StyledSelect>
+
+      <label htmlFor="hobbies">Hobbies:</label>
+      <StyledInput
+        type="text"
+        id="hobbies"
+        name="hobbies"
+        value={formData.hobbies}
+        onChange={handleInputChange}
+      />
+
+      <label htmlFor="city">City:</label>
+      <StyledInput
+        type="text"
+        id="city"
+        name="city"
+        value={formData.city}
+        onChange={handleInputChange}
+      />
+
+      <h3>Contact Options:</h3>
+      {formData.contactOptions.map((contactOption) => (
+        <Fragment key={contactOption.type}>
+          <label htmlFor={contactOption.type}>{contactOption.name}</label>
+          {numberTypes.includes(contactOption.type) ? (
+            <StyledInput
+              type={contactOption.type}
+              id={contactOption.type}
+              name={contactOption.type}
+              placeholder={`Insert ${contactOption.name} Number`}
+              value={contactOption.number}
+              onChange={handleContactOptionChange}
+            />
+          ) : (
+            <StyledInput
+              type={contactOption.type}
+              id={contactOption.type}
+              name={contactOption.type}
+              placeholder={`Insert ${contactOption.name} Address`}
+              value={contactOption.address}
+              onChange={handleContactOptionChange}
+            />
+          )}
+        </Fragment>
```

Comment:
This may have been not easy to implement. Cool! 👍 But: you hand over to your inputs types like mobile, signal, telegram etc. Instead you should work with the types an input element really has. [Check MDN for input types that are really available. ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input)